MiniSpace.Web/src/MiniSpace.Web/Shared/MainLayout.razor

```diff
@@ -49,12 +60,14 @@
                                          Click="@(() => NavigationManager.NavigateTo("account"))" />
                     <RadzenPanelMenuItem Text="Events" Icon="event"
                                          Click="@(() => NavigationManager.NavigateTo("events"))" />
+                    <RadzenPanelMenuItem Text="Friends" Icon="group"
+                             Click="@(() => NavigationManager.NavigateTo("friends"))" />
```

Comment:
This is technically-practically the issue related to previous one:

Currently I will not resolve this one. Please do not forget about that in the next iteration. This more like 'carving up' with the identity service and the local storage with the students service.